core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -87,8 +87,46 @@
               </div>
             </div>
           </div>
-        </div>
+          <div class="text-right">
+            <button class="btn btn-success" style="width: 15em;" data-toggle="modal" data-target="#myModal" onclick="load()">Preview Summary Tite</button>
```

Comment:
Don't use onclick, data-toggle, data-target. Use ng-click and do it the Angular way, similar to how other modals in the site are handled.